### PR TITLE
remove fakeAuthProvider and replace with one from bcl

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/neworg/routes.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import request from 'supertest';
+import FakeAuthProvider from 'buying-catalogue-components/app/common/test/fakeAuthProvider';
```
Comment (alibombaye) [inline app/pages/neworg/routes.test.js:2]: can we do something to tidy this path up a little?

Something like 
import FakeAuthProvider from 'buying-catalogue-components/library';

Comment (kat1906) [reply app/pages/neworg/routes.test.js:2]: this is the path - would have been easier if you commented on the bcl when I put it in 😄. I will move it out of app and into a folder called library but I was going to have all in their own folder with index file

Comment (alibombaye) [reply app/pages/neworg/routes.test.js:2]: yeah that's fine. In BCL didn't realise the import would look like this 😂

Comment (kat1906) [reply app/pages/neworg/routes.test.js]: fixed it - much better

Comment (alibombaye) [reply app/pages/neworg/routes.test.js]: 😍

**Pull request merged**